app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
```

Comment:
an error message is strange. You perform checks for one visitor, so if he is not vaccinated why the message is "all friends". Who are these friends?